charts/argo-cd/README.md

```diff
@@ -316,6 +316,8 @@ NAME: my-release
 | repoServer.clusterRoleRules.rules | list | `[]` | List of custom rules for the Repo server's Cluster Role resource |
 | repoServer.containerPort | int | `8081` | Configures the repo server port |
 | repoServer.containerSecurityContext | object | `{}` | Repo server container-level security context |
+| repoServer.copyutil | object | `{"resources":{}}` | Init Container to copy argocd binary |
```

Comment:
Not a blocker, but probably we could drop this line due being implicitly documented in next one? :thinking:

Reply:
I removed it (automatically with `helm-docs`)